src/index.html

```diff
@@ -6,22 +6,359 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Catalog</title>
-    <link
-      rel="preconnect"
-      href="https://fonts.gstatic.com"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet"
-    />
+    <title>Catalog Page</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The stylesheet link is pointing to a SCSS file (`index.scss`). HTML cannot directly use SCSS files; they need to be compiled into CSS first. Please compile your SCSS to CSS and update the `href` attribute to point to the resulting CSS file.